core/src/price_estimation.rs

```diff
@@ -37,7 +37,14 @@ type Tokens = BTreeMap<TokenId, Option<TokenInfo>>;
 /// tokens included in the current orderbook.
 #[cfg_attr(test, mockall::automock)]
 pub trait PriceEstimating {
+    /// Retrieves price estimates for all token in the specified orders on a
+    /// best-effort basis. Critically, it is not considered an error to not be
+    /// able to retrieve a price estimate for a given token.
     fn get_token_prices<'a>(&'a self, orders: &[Order]) -> BoxFuture<'a, Tokens>;
+
+    /// Retrieves a price estimate for ETH in OWL.
+    #[allow(clippy::needless_lifetimes)]
+    fn get_eth_price<'a>(&'a self) -> BoxFuture<'a, Option<u128>>;
```

Comment:
This is the price of 1 ether (10e18 wei) in owl atoms, right? I'd put this in the comment for clarity.